test/jdk/java/lang/String/StringRepeat.java

```diff
@@ -27,21 +27,33 @@
  * @run main/othervm -Xmx2g StringRepeat
```

Comment:
So, does `-Xmx2g` make sense in this configuration now?

Reply:
No, not really. The first test case instance should require way less than 1G of memory, so I will just remove the heap configuration for that instance.